lib/Caxy/HtmlDiff/HtmlDiffConfig.php

```diff
@@ -468,6 +473,26 @@ public function getCacheProvider()
         return $this->cacheProvider;
     }
 
+    /**
+     * @return bool
+     */
+    public function isPurifierEnabled()
+    {
+        return $this->purifierEnabled;
+    }
+
+    /**
+     * @param bool $purifierEnabled
+     *
+     * @return $this
+     */
+    public function setPurifierEnabled($purifierEnabled = true)
```

Comment:
Since this version of the library (master van v0.1.10 and up) only works on PHP7.3 and up, you can remove the docblock and strict type-hint this method declaration